src/dev/impl/DevToys/ViewModels/Tools/EncodersDecoders/JwtDecoderEncoder/JwtDecoderEncoderViewModel.cs

```diff
@@ -0,0 +1,653 @@
+﻿#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Composition;
+using System.IO;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using DevToys.Api.Core;
+using DevToys.Api.Core.Settings;
+using DevToys.Core;
+using DevToys.Helpers;
+using DevToys.Helpers.JsonYaml;
+using DevToys.Models;
+using DevToys.Models.JwtDecoderEncoder;
+using DevToys.UI.Controls;
+using Microsoft.IdentityModel.Tokens;
+using Microsoft.Toolkit.Mvvm.Messaging;
+using Microsoft.UI.Xaml.Controls;
+using Org.BouncyCastle.Crypto;
+using Org.BouncyCastle.Crypto.Parameters;
+using Org.BouncyCastle.Math;
+using Windows.UI.Xaml;
+using PemReader = Org.BouncyCastle.OpenSsl.PemReader;
+
+namespace DevToys.ViewModels.Tools.EncodersDecoders.JwtDecoderEncoder
+{
+    [Export(typeof(JwtDecoderEncoderViewModel))]
```

Comment:
Is it normal that we Export this class? I don't see it imported anywhere.
Also, I see we inherit, which makes me think perhaps we should name it `JwtDecoderEncoderViewModelBase` and make it `abstract` ?